Source/Core/DolphinQt/DolphinEmu.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.cs.allow-jit</key>
+	<true/>
+	<key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
just curious, what is `com.apple.security.cs.disable-library-validation` required for?

Reply:
oh, i didn't see that vulkan is added in this PR .. as a precompiled dynamic library :/

Reply:
This is no longer needed since I moved the project to utilize deep code signing(previously I had issues with unsigned library dependencies). I will remove that entitlement in the next push.

Reply:
Looks like I will have to add this entitlement back, as removing this breaks adhoc signed builds since adhoc signed frameworks fail library validation.